### PR TITLE
WEBSITE-265: Some follow-ups

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

File: _importer/README.md [hunk 1]
```diff
@@ -19,11 +19,78 @@ old to new URLs.
     > bundle exec ./crawler.rb -u http://in.relation.to -p ".*\.lace" -o posts.pstore | tee crawl.log
 
     # run the importer (creates erb files, Apache redirects and Disqus WXR import file)
-    > bundle exec ./importer.rb -s posts.pstore -o ../posts
+    > bundle exec ./importer.rb -s posts.pstore -o ../posts -rf ../redirects/.htaccess_posts
 
     # for experiments when you don't want to download images (-ni) and assets (-na)
     > bundle exec ./importer.rb -s posts.pstore -o ../posts -ni -na
 
+## Creating re-directs
```
Comment (hferentschik) [inline _importer/README.md]: ahh, thanks for the docs :-)  I was about to ask

File: redirects/.htaccess_misc [hunk 2]
```diff
@@ -6,6 +6,16 @@ RewriteRule ^Bloggers/Everyone$ / [R=301,L,E=nocache:1]
...
 RewriteRule ^service/Feed/atom/Area/Bloggers(/Comments/.*)?$ /blog.atom [R=301,L,E=nocache:1]
 
+# Jay's categories
+
+RewriteRule ^Bloggers/Aerogear$ /aerogear/ [R=301,L,E=nocache:1]
+RewriteRule ^Bloggers/Interop$ /seam/ [R=301,L,E=nocache:1]
+RewriteRule ^Bloggers/Tweetstream$ /rich-faces/ [R=301,L,E=nocache:1]
+RewriteRule ^Bloggers/Mobile$ /rich-faces/ [R=301,L,E=nocache:1]
+RewriteRule ^Bloggers/Openshift$ /jay-balunas/ [R=301,L,E=nocache:1]
+RewriteRule ^Bloggers/Releases$ /seam/ [R=301,L,E=nocache:1]
```
Comment (emmanuelbernard) [inline redirects/.htaccess_misc]: OpenShift redirecting to Jay?
Releases redirecting to Seam? That does not look right.

Comment (gunnarmorling) [inline redirects/.htaccess_misc]: Check out the contents behind these URLs on the old blog.

**Comment by hferentschik** [comment]
Merged.

File: redirects/.htaccess_misc (hunk 2 above)
Comment (emmanuelbernard) [inline redirects/.htaccess_misc]: ok some additional comment for the context would be useful one year from now :)

**Pull request closed without merging**